tests/unit/test_charm.py

```diff
@@ -852,14 +861,38 @@ def test_on_update_status(harness):
 
         # Test after the cluster was initialised, but with the unit in a blocked state.
         with harness.hooks_disabled():
+            harness.set_leader()
             harness.update_relation_data(
                 rel_id, harness.charm.app.name, {"cluster_initialised": "True"}
             )
         harness.charm.unit.status = BlockedStatus("fake blocked status")
         harness.charm.on.update_status.emit()
         _set_primary_status_message.assert_not_called()
 
+        # Test the point-in-time-recovery fail.
+        with harness.hooks_disabled():
+            harness.update_relation_data(
+                rel_id,
+                harness.charm.app.name,
+                {
+                    "cluster_initialised": "True",
+                    "restoring-backup": "True",
+                    "restore-to-time": "True",
```

Comment:
Minor change: set a "valid" value instead of a boolean string.

Reply:
Done